release-notes/6.0/preview/api-diff/preview5/.Net/6.0-preview5_Microsoft.Win32.SafeHandles.md

```diff
@@ -0,0 +1,27 @@
+# Microsoft.Win32.SafeHandles
+
+``` diff
+ namespace Microsoft.Win32.SafeHandles {
++    public abstract class SafeNCryptHandle : SafeHandleZeroOrMinusOneIsInvalid {
++        protected SafeNCryptHandle();
```

Comment:
What's new here?  Most of this stuff previously existed.

Reply:
It seems like we added openssl and cng dlls to the ref pack cc @ericstj @ViktorHofer

Reply:
https://github.com/dotnet/runtime/pull/51853/files#diff-dd4eb818f3a25bb1e97606532166e6b30deecb4f91580b672a5527200f233795L153
https://github.com/dotnet/runtime/commit/7b575d777bb9eda5a5d6107e812e1a154fce6739#diff-dd4eb818f3a25bb1e97606532166e6b30deecb4f91580b672a5527200f233795L154

here is the change i think that added these assemblies.

Reply:
The implementation was in the shared framework already, but the reference assembly was OOB because of single-platform support.  Now that we have the platform analyzer stuff we got rid of the package, so this is probably showing up as being "new in the shared framework reference API set"

Reply:
If we have a reasonable way of leaving all of the SafeNCrypt*Handle types out, I think that'd be best.  It's not "new API" so much as "we changed some layout".

Reply:
yeah i can go ahead and remove them.